### DISCUSSION TITLE
{"code":500,"message":"Loading storage, please wait","data":null}

### DISCUSSION THREAD

**Discussion by typhon1144** [AlistGo/alist#2417] [Q&A]
不知道为什么突然一下localhost:5244打不开了出现这个{"code":500,"message":"Loading storage, please wait","data":null}在浏览器里面

**Comment by typhon1144** [comment]
已经知道原因载入了太多文件导致的

**Reply by yuedumingz** [reply]
我也遇到了，知道怎么解决吗

**Reply by yuedumingz** [reply]
已解决，等一会自己就好了
